handlers.py

```diff
@@ -40,6 +47,37 @@ def handle_get_ssid(server, body, addr):
     server.send(json.dumps(resp).encode('utf-8'), addr)
 
 
+def register_car(userID):
+    """
+    Registers the car with the central server. Retries forever until the wanted
+    ACK is received. Returns the carID received in the ACK.
+
+    Arguments:
+    userID -- The ID of the user that is registering the car
+    """
+    server = Server(WLAN_IP, PORT, 60)
+    server.add_handler(MsgType.ACK, None)
```

Comment:
I think this line is unnecessary.

Reply:
It actually doesn't receive if I don't add a handler because the `receive` function checks if the type is registered in the handlers of the Server instance, so it ends up just sending an ERROR response

Reply:
While I was driving, I was thinking about just changing the receive function to do the basic receive and put the type handling in the `receive_forever`

Reply:
That makes sense to me.